internal/indexmanagement/scripts.go

```diff
@@ -660,14 +662,19 @@ function updateWriteIndex() {
 `
 
 const rolloverScript = `
-set -euo pipefail
+set -eo pipefail
 source /tmp/scripts/indexManagement
 
 decoded=$(echo $PAYLOAD | base64 -d)
 
 # need to get a list of all mappings under ${POLICY_MAPPING}, drop suffix '-write' iterate over
 writeAliases="$(getWriteAliases "$POLICY_MAPPING")"
 
+if [ -v $writeAliases ]; then
```

Comment:
Shouldn't this also be the "zero test" (`-z`) like in the other script?

```suggestion
if [ -z $writeAliases ]; then
```

Reply:
This must have been from another one of my trials (along with omitting `nounset` from the options). It has no special purpose so I'll just revert it :)